src/patternfly/components/Banner/banner.scss

```diff
@@ -20,9 +20,7 @@
   --#{$banner}--link--Color: var(--#{$banner}--Color);
```

Comment:
If you hover over the "inline link button" link in the blue banner, the text color changes to blue.

I think you can just make these changes:
* Remove this `:hover` block for inline buttons - we should be able to handle this with the next point, and just set button vars. https://github.com/patternfly/patternfly/blob/728f578a7f74398df41f6920e1e4ae28976bf0b8/src/patternfly/components/Banner/banner.scss#L167-L169
* On this line, change `--#{$button}--m-link--m-inline--hover--Color` to `--#{$button}--m-link--hover--Color`. The previous var no longer exists, and this should set the button's hover color - https://github.com/patternfly/patternfly/blob/728f578a7f74398df41f6920e1e4ae28976bf0b8/src/patternfly/components/Banner/banner.scss#L162